test/additionalTests/LP9test/testLP9.m

```diff
@@ -0,0 +1,87 @@
+% The COBRAToolbox: Lp9.m
+%
+% Purpose:
+%     - test if bug fix in LP9 affects the results for test model (without
+%     coupling constraints)
+%
+% Authors:
+%     - fix compatibility for models with coupling constraints: Agnieszka
+%     Wegrzyn 2016/03/06
+%
+
+global CBTDIR
+
+% define the features required to run the test
+requiredToolboxes = {};
+
+requiredSolvers = {'glpk'};
```

Comment:
Eventually, it might be best to use `gurobi` for the test.

Reply:
currently, I do not have Gurobi installed so I cannot test with it

Reply:
Ok, I installed Gurobi and edited the test to work with it.